guillotina/content.py

```diff
@@ -569,7 +571,13 @@ def iter_schemata(obj) -> Iterator[Interface]:
 
 @profilable
 async def create_content_in_container(
-    parent: Folder, type_: str, id_: str, request: IRequest = None, check_security=True, **kw
+    parent: Folder,
+    type_: str,
+    id_: str,
+    request: IRequest = None,
+    check_security: bool = True,
+    check_constraints: bool = True,
```

Comment:
just last comment before the merge, to be sure: @bloodbare did you prefer `check_constrains=False` by default on `create_content_in_container`?

Reply:
Default should be true!

Reply:
cool! then we're good to merge !